scripts/functions_delete_package.py

```diff
@@ -0,0 +1,109 @@
+"""
+Copyright 2018 IBM Corporation
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import os, json, sys, argparse, requests, zipfile, base64
+from requests.packages.urllib3.exceptions import InsecureRequestWarning
+from cfgCommons import Cfg
+from wawCommons import setLoggerConfig, getScriptLogger, getFilesAtPath, openFile, getRequiredParameter, getOptionalParameter, getParametersCombination, convertApikeyToUsernameAndPassword
+import urllib3
+import logging
+
+
+logger = getScriptLogger(__file__)
+
+def isActionNotSequence(action):
+    for annotation in action['annotations']:
+        if annotation['key'] == 'exec' and annotation['value'] == 'sequence': return True;
```

Comment:
I would rather check if keys are present to avoid exceptions. Logic seems to be switched.

Reply:
Fixed.